sync.py

```diff
@@ -14,7 +14,9 @@
 p4.user = CONFIG.P4_USER
 p4.password = CONFIG.P4_PASSWD
 p4.connect()
-p4.run_login()
+if not CONFIG.P4_USE_TICKET:
+    p4.run_login()
```

Comment:
can you move `p4.password` to here?
imo, ticket base auth doesn't need password :)

Reply:
You are right when we have a `.p4tickets`. However, `p4.password` can store a ticket also. So I prefer to keep the `p4.password` for a convenient reason. Make sense?